source/index.html.md

```diff
@@ -540,26 +527,103 @@ Exactly one payment method must be used.
+    a later transaction where the payment credential is not provided by the
+    cardholder but collected from (encrypted) storage.
+    <br />
+    <code>storing</code>: The payment credential will be stored; may only be
+    stored if the authorization is approved.
+    <br />
+    <code>using</code>: The payment credential was already stored and are now
+    being used.
+    <br />
+    If <code>initiator</code> is <code>merchant</code> then
+    <code>storing</code> is invalid and <code>using</code> is assumed.
+    For a first-in-series authorization, <code>storing</code> is assumed.
+
+    <div class="type">Optional</code>
+  </dd>
+
+  <dt>initiator
+    <span class="type">(<code>cardholder</code>|<code>merchant</code>)</span>
+  </dt>
+  <dd>
+    The initiator of the authorization. An authorization is initiated by the
+    cardholder if the cardholder decided the transaction should be created.
+    This is regardless of whether stored payment credential is being used.
+    <br />
+    For compliance reasons there should be a previous approved transaction
+    marked with <code>storing</code> before <code>initiator</code> may be
+    <code>merchant</code>.
+    For subsequent-in-series authorizations, <code>merchant</code> is assumed;
+    otherwise, <code>cardholder</code> is assumed.
+
+    <div class="type">Optional</div>
+  </dd>
+
   <dt>ip
     <span class="type">[0-9.a-fA-F:]{3,45}</span>
   </dt>
   <dd>
     Cardholder's IP address. It must be a valid v4 or v6 address.
     <div class="type">Optional<br></div>
   </dd>
-  <dt>recurring
-    <span class="type">(true|false)</span>
+
+  <dt>
+    series[type]
+    <span class="type">(<code>recurring</code>|<code>unscheduled</code>)</span>
   </dt>
   <dd>
-    Must be <code>true</code> for recurring payments.
-    <div class="type">Optional</div>
+    Indicate the type of series.
+    <br />
+    <code>recurring</code>: A series of transactions where the cardholder has
+    explicitly agreed that the merchant will repeatedly charge the cardholder at
+    regular, predetermined intervals.
+    <br />
+    <code>unscheduled</code>: A series of transactions where the cardholder has
+    explicitly agreed that the merchant will repeatedly charge the cardholder at
+    unknown times, e.g. based on cardholder usage.
+    <br />
+    Although the amount may vary among transactions in any series, the amount
+    of any subsequent-in-series transaction can not exceed the amount of the
+    authentication of the first-in-series authorization.
+
+    <div class="type">Conditional. Cannot be present if <code>series[previous][id]</code> is present.</div>
+  </dd>
+
+  <dt>
+    series[previous][id]
+    <span class="type">[:UUIDv4:]</span>
+  </dt>
+  <dd>
+    The Clearhaus authorization ID as a reference to the latest approved
+    authorization in the series. The authorization is processed as a
+    first-in-series if not populated while <code>series[type]</code> is
+    populated.
```

Comment:
I think something is wrong with this sentence?

"a is processed as b if not populated while c is populated"

I'm confused :smile:

Reply:
I see that this became confusingly minimalistic.

Let me try to "de-confuse" 😄

> "a is processed as b if <this field, implicitly> [is] not populated while [in the sense "but"] c is populated"

I'll try to fix the wording 🙂

Reply:
Fixed by https://github.com/clearhaus/gateway-api-docs/pull/111/commits/e4a6aebaf5e53b32346e1cb0199460b395865f85#diff-d30d91dbcd59693b69f9242acb4630c5R647-R650